src/udev/udev-event.c

```diff
@@ -945,8 +944,7 @@ static int udev_event_on_move(UdevEvent *event) {
         sd_device *dev = event->dev;
         int r;
 
-        if (event->dev_db_clone &&
-            sd_device_get_devnum(dev, NULL) < 0) {
+        if (sd_device_get_devnum(dev, NULL) < 0) {
                 r = device_copy_properties(dev, event->dev_db_clone);
```

Comment:
Same here, how do we konw dev_db_clone is nonnull?

Reply:
On 'move' event, `dev_db_clone` is always created.

Reply:
We'll get an assert failure is this is not satisfied. Should be OK to merge.